pom.client.xml

```diff
@@ -162,18 +162,6 @@
         <version>${azure-client-runtime.version}</version>
       </dependency>
 
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-common</artifactId>
-        <version>${azure-common.version}</version>
-      </dependency>
-
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-common-auth</artifactId>
-        <version>${azure-common-auth.version}</version>
-      </dependency>
-
```

Comment:
Not sure why this is removed?

Reply:
It was added twice, so there was a warning. It still exists below here: https://github.com/Azure/azure-sdk-for-java/blob/db2c4ad69ef53eff0554fb8c51ad9dcdf1845fdf/pom.client.xml#L171-L181

Reply:
👍